tests/pytests/test_matrix.py

```diff
@@ -267,6 +267,28 @@ def test_back_transform():
     assert compare_arrays(transformer.nph[0] @ original.nph[0] @ (transformer.nph[1]).T, transformed.nph[0]) 
     assert compare_arrays(transformer.nph[1] @ original.nph[1] @ (transformer.nph[0]).T, transformed.nph[1])
 
+def test_get_matrix():
+    # Use get_matrix to extrct a block of a non-totally symmetric matrix.
```

Comment:
```suggestion
    # Use get_matrix to extract a block of a non-totally symmetric matrix.
```
feel free to defer fix to save CI